README.md

```diff
@@ -1,12 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# BOOKSTORE
 
-> In this project we will prepare a Book store for adding books usinf react and redux liberary.
+> In this Book Store project:
```

Comment:
- [ ] Kindly deploy the project online and include the livelink in the readme.

Reply:
Done